src/sentry/monitors/logic/mark_failed.py

```diff
@@ -140,6 +140,7 @@ def mark_failed_threshold(
 
         starting_checkin = previous_checkins[0]
 
+        incident: MonitorIncident | None
```

Comment:
Would it be a bit clearer if this was defined outside the if statement?

Reply:
potentially -- though I put it next to the first assignment so it didn't get potentially orphaned